docker/test/fasttest/run.sh

```diff
@@ -158,6 +158,7 @@ function run_cmake
         "-DENABLE_NURAFT=1"
         "-DENABLE_JEMALLOC=1"
         "-DENABLE_REPLXX=1"
+        "-DENABLE_RUST=0"
```

Comment:
Why do you need this? Just don't install Rust and it will set `ENABLE_RUST=0` automatically, won't it?

Reply:
Removed the flag, everything works without it in fasttest